packages/vue/src/auth/service.ts

```diff
@@ -152,4 +152,20 @@ export class FronteggAuthService implements FronteggPluginService {
       }
     }
   };
+
+
+  getCurrentRoute = () => {
+    let route: string = '/';
+    if (this.router?.currentRoute?.path) {
+      // vue-router v3
+      route = this.router?.currentRoute?.path;
+    }
+    // @ts-ignore vue-router v4
+    if (this.router?.currentRoute?.value?.path) {
+      // @ts-ignore vue-router v4
+      route = this.router?.currentRoute?.value?.fullPath;
```

Comment:
why do we check if `this.router?.currentRoute?.value?.path` and then return `this.router?.currentRoute?.value?. fullPath`